baselines/Horizontal_XGBoost/Horizontal_XGBoost/conf/Centralized Baseline.yaml

```diff
@@ -0,0 +1,47 @@
+centralized: True
```

Comment:
Could you rename this file so there is not a space ? e.g. rename to `centralized_baseline.yaml`?